UtopiaCity/Controllers/Life/WeatherController.cs

```diff
@@ -36,7 +36,7 @@ public async Task<UtopiaCity.Models.Airport.WeatherReport> GetFlightWeatherRepor
                 var data = await WeatherReportApi.GetReportClosestToDateAsync(date);
                 if (data != null)
                 {
-                    return _mapper.Map<UtopiaCity.Models.Airport.WeatherReport>(data);
+                    return _mapper.Map<Models.Airport.WeatherReport>(data);
```

Comment:
Maybe even just WeatherReport would be better

Reply:
ou , its not mine ) but ok , will change)